UnitySetup/UnitySetup.psm1

```diff
@@ -125,8 +125,16 @@ class UnityProjectInstance {
         $projectSettingsFile = [io.path]::Combine($path, "ProjectSettings\ProjectSettings.asset")
         if (!(Test-Path $projectSettingsFile)) { throw "Project is missing ProjectSettings.asset" }
 
-        $prodName = ((Get-Content $projectSettingsFile -Raw | ConvertFrom-Yaml)['playerSettings'])['productName']
-        if (!$prodName) { throw "ProjectSettings is missing productName" }
+        try { 
+            $prodName = ((Get-Content $projectSettingsFile -Raw | ConvertFrom-Yaml)['playerSettings'])['productName']
+            if (!$prodName) { throw "ProjectSettings is missing productName" }
+        }
+        catch {
+            Write-Warning -Message "An Exception was caught!"
```

Comment:
I think it's reasonable to print the exception, but since we know the issue you're seeing will be common, we should point that out with a message something akin to, "Could not read $projectSettingsFile, have you set your meta files to "Force Text"".